charts/consul/test/unit/server-config-configmap.bats

```diff
@@ -658,6 +658,43 @@ load _helpers
   [ "${actual}" = "true" ]
 }
 
+@test "server/ConfigMap: doesn't set Vault Namespace in connect CA config when global.secretsBackend.vault.vaultNamespace is blank in values.yaml" {
```

Comment:
Re: other comment, can we also check here for conditional overrides when both are set?